operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -309,16 +309,33 @@ spec:
 # The QPS value of the Kubernetes client.
 #    ---
 #    qps: 175
+
+##########
+#  ---
+#  cache_config:
 #
-# Flag to use a Kubernetes cache for watching changes and updating pods and controllers data asynchronously.
-# Caution: Kubernetes cache is not compatible with reduced permissions scenearios.
+# Flag to use a cache for watching changes in Istio / Kubernetes. Note, enabling cache only
+# caches projects / namespaces. To cache more objects, enable istio_objects and/or kubernetes_objects
+# to cache more items.
+# Caution: Cache is not compatible with reduced permissions scenarios. Performance issues may arise
+# on the Kubernetes registry if too many namespaces are cached.
 #    ---
 #    cache_enabled: false
```

Comment:
maintain alphabetical ordering. Doc the settings in the order:
* cache_duration
* cache_enabled
* istio_objects
* kubernetes_objects